core/src/services/split-view.js

```diff
@@ -0,0 +1,228 @@
+import { LuigiConfig, LuigiElements } from '../core-api';
+import { Navigation } from '../navigation/services/navigation';
+import { Iframe } from '../services';
+import {
+  GenericHelpers,
+  IframeHelpers,
+  RoutingHelpers
+} from '../utilities/helpers';
+
+class SplitViewSvcClass {
+  constructor() {
+    this.splitViewValues;
+    this.storedSplitViewValues;
+  }
+
+  getSplitViewContainer() {
```

Comment:
As we are calling all these methods with `SplitViewSvc` maybe we don't need to add 'SplitView` to the method names. What do you think?